### DISCUSSION TITLE
[Projects Beta] Sort views by dragging and dropping view tabs

### DISCUSSION THREAD

**Discussion by it-mse** [community/community#6266] [Projects and Issues] [answered]
After creating a view, I would like to drag the tab that represents that view to sort it manually.

**Comment by barelyknown** [comment, marked as answer]
Duplicate of [this discussion](https://github.com/github/feedback/discussions/4803).

**Comment by azenMatt** [comment]
Hi @it-mse @barelyknown  - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄
